.storybook/stories/documentation/Blocks.mdx

```diff
@@ -14,7 +14,11 @@ Each block has the following common properties:
 
 `visible?: 'all' | 'sm' | 'md' | 'lg' | 'xl'` — Sets the screen size to start block display from
 
-`resetPaddings: boolean` — Allows resetting top and bottom margins standard for all blocks
+`resetPaddings: boolean` — Allows resetting top and bottom margins standard for all blocks. **Deprecated**, use `indentTop, indentBottom` instead
+
+`indentTop` - block indentation at the top, default size `l`, examples you can see [here](?path=/story/blocks-cardlayout--with-custom-indents)
```

Comment:
didn't you think about lile
```
indent?: {
  top?: number
  bottom?: number
}
```

Reply:
It seems to me that these types of records are the same. Maybe there is some reason to write through an object ?

Reply:
yes, object notation gives less non-required top level props, in editor, for example, we show all block properties, so using one property instead of two can to reduce the visual noise

Reply:
refactor